README.md

```diff
@@ -5,14 +5,12 @@ bluesky
 
 a Python data collection interface for experimental science
 
-See [documentation](http://nsls-ii.github.io/bluesky).
+See [documentation](http://bluesky.github.io/bluesky).
 
-[Report an issue with bluesky](https://github.com/NSLS-II/bluesky/issues/new)
+[Report an issue with bluesky](https://github.com/bluesky/bluesky/issues/new)
 
 ## Conda Recipes
 
-Install the most recent tagged build: `conda install bluesky -c lightsource2-tag`
+Install the most recent build: `conda install bluesky -c nsls2force`
```

Comment:
I think this was meant to be `nsls2forge`? :)

Reply:
It is, thanks for catching the typo I missed!

Reply:
Good catch!